src/controllers/mine.py

```diff
@@ -0,0 +1,23 @@
+import pymongo
+import json
+import sys
+sys.path.append('..')
+from controllers.block import Block
+from utils.dbProvider import DBProvider
+
+
+class Mine:
```

Comment:
A better use of the problem language would be to call this class BlockModel.
We will eventually have more than one BlockModel for one single Mine, so they should be separate concepts.

Reply:
fixed